inc/machine.inc

```diff
@@ -0,0 +1,19 @@
+; Error return value for kernal calls which indicate
+; the machine does not have the necessary capabilities
+ERROR_MACHINE_TYPE = 40
+
+; Machine has detected a 65C816 processor
+MACHINE_TYPE_C816       = %10000000
+; Machine has flat memory model at >= $010000
+MACHINE_TYPE_FLAT24     = %01000000
```

Comment:
Technically it's not a flat memory model, as the program counter wraps at the page boundary.